Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/App.java

```diff
@@ -19,10 +20,15 @@ public void onCreate() {
                 .repositoryModule(new RepositoryModule())
                 .stringProviderModule(new StringProviderModule(this.getApplicationContext()))
                 .validationModule(new ValidationModule())
+                .filePickerModule(new FilePickerModule())
                 .build();
     }
 
     public AppComponent getAppComponent() {
         return appComponent;
     }
+
+    public void setComponent(AppComponent appComponent) {
```

Comment:
Could it be other solution for mock tests then changing code? Could you try with reflection or power mockito to mock app component? Power mockito allows to mock private fields I think. What do you think? http://www.johnmullins.co/blog/2015/02/15/beginners-guide-to-using-mockito-and-powermockito-to-unit-test-java/